app/web/src/newhotness/Explore.vue

```diff
@@ -802,49 +795,117 @@ const sortedAndGroupedComponents = computed(() => {
 // THE SEARCH BAR AND FILTERING
 const searchString = ref<string>("");
 const debouncedSearchString = ref<string>("");
-const computedSearchString = computed(() => debouncedSearchString.value);
 
 // send this down to any components that might use it
-provide("SEARCH", computedSearchString);
-
-watch(
-  () => [debouncedSearchString.value, componentList.value],
-  () => {
-    if (!debouncedSearchString.value) {
-      filteredComponents.splice(0, Infinity, ...componentList.value);
-      return;
+provide("SEARCH", debouncedSearchString);
+
+/**
+ * Search string, split into terms.
+ *
+ * MyComponent vpcId:vpc-123 AWS::EC2::Subnet parses to:
+ *
+ *     {
+ *       attrSearchTerms: [ { key: "vpcId", value: "vpc-123" } ],
+ *       knownSearchTerms: [ { key: "schema", value: "AWS::EC2::Subnet" } ],
+ *       fuzzyTerms: [ "MyComponent", "AWS::EC2::Subnet" ]
+ *     }
+ */
+const debouncedSearchTerms = computed(() => {
+  const attrSearchTerms = new Array<{ key: string; value: string }>();
+  const knownSearchTerms = new Array<{ key: "schema"; value: string }>();
+  const fuzzyTerms = new Array<string>();
+  for (const term of debouncedSearchString.value.split(/\s+/g)) {
+    // Treat a:b as a key-value pair
+    const colon = term.indexOf(":");
+    if (colon >= 0 && term[colon+1] !== ":") { // Don't treat :: as a key-value pair
+      const key = term.slice(0, colon);
+      const value = term.slice(colon + 1);
+      if (key === "schema") {
+        knownSearchTerms.push({ key, value });
+      } else {
+        attrSearchTerms.push({ key, value });
+      }
+    } else {
+      fuzzyTerms.push(term);
     }
+  }
+  return { attrSearchTerms, knownSearchTerms, fuzzyTerms };
+});
 
-    const searchTerm = debouncedSearchString.value.trim();
+/**
+ * Components list filtered by the advanced search terms (by querying bifrost).
+ *
+ * Should only be used by filteredComponents.
+ */
+const attrSearchedComponentIds = computedAsync(async () => {
+  const { attrSearchTerms } = debouncedSearchTerms.value;
+  if (attrSearchTerms.length === 0) {
+    return undefined;
+  }
 
-    // Check if the search term contains "schema:" queries
-    const schemaMatches = searchTerm.match(/schema:([^\s]+)/gi);
-    if (schemaMatches) {
-      const schemaNames = schemaMatches.map((match) =>
-        match.substring(7).trim().toLowerCase(),
-      );
+  const result = await bifrostQueryAttributes({
```

Comment:
We need to make a change here:
1. Either, wrap this in useQuery with a key that the web worker can bust when AV changes come across.
2. Add something inside this computed that will trigger a re-query when AV changes happen.

The first is easier